src/integration/java/io/pinecone/integration/dataPlane/UpsertAndQueryServerlessTest.java

```diff
@@ -23,25 +23,23 @@
 import static io.pinecone.commons.IndexInterface.buildUpsertVectorWithUnsignedIndices;
 import static io.pinecone.helpers.AssertRetry.assertWithRetry;
 import static io.pinecone.helpers.BuildUpsertRequest.*;
-import static io.pinecone.helpers.IndexManager.*;
 import static org.junit.jupiter.api.Assertions.*;
 
 public class UpsertAndQueryServerlessTest {
-
+    private static final TestIndexResourcesManager indexManager = TestIndexResourcesManager.getInstance();
     private static Index index;
     private static AsyncIndex asyncIndex;
+    private static int dimension;
+    private static final String namespace = RandomStringBuilder.build("ns", 8);
```

Comment:
This will change w/my PR

Reply:
Answer here: https://github.com/pinecone-io/pinecone-java-client/pull/99/files#r1576461309